samigo/samigo-app/src/webapp/js/samigo-global.js

```diff
@@ -86,8 +87,11 @@ function whichradio(el) {
 		$('input[type=radio]',parentTable).not(el).each(function(){
 			var id = $(this).prop('id');
 			if(id.indexOf(colId) !== -1 && $(this).is(':checked')) {
-				$(this).prop('checked',false);
+				alert("You are only allowed one selection per column, please try again.");
```

Comment:
This string was previously never internationalized. It was removed in https://jira.sakaiproject.org/browse/SAM-2926, causing a regression that user input was invalid. I merely restored the string. I certainly agree that it should be internationalized!